src/main/webapp/resources/js/pages/user/components/UserProjectsPage.jsx

```diff
@@ -1,10 +1,81 @@
 import React from "react";
+import { useParams } from "react-router-dom";
+import { notification, Space, Switch, Table, Typography } from "antd";
+import { setBaseUrl } from "../../../utilities/url-utilities";
+import { formatDate } from "../../../utilities/date-utilities";
+import { useUpdateProjectSubscriptionMutation } from "../../../apis/projects/project-subscriptions";
+import { PagedTableProvider } from "../../../components/ant.design/PagedTable";
+import {
+  PagedTable,
+  PagedTableContext,
+} from "../../../components/ant.design/PagedTable";
 
 /**
  * React component to display the user projects page.
  * @returns {*}
  * @constructor
  */
 export default function UserProjectsPage() {
-  return <div>User Projects Page</div>;
+  const { userId } = useParams();
+  const [updateProjectSubscription] = useUpdateProjectSubscriptionMutation();
+  const columns = [
+    {
+      title: i18n("UserProjectsPage.table.projectId"),
+      dataIndex: "projectId",
+      key: "projectId",
+    },
+    {
+      title: i18n("UserProjectsPage.table.projectName"),
+      dataIndex: "projectName",
+      key: "projectName",
+      render: (text, record) => (
+        <a href={setBaseUrl(`projects/${record.projectId}`)}>{text}</a>
+      ),
+    },
+    {
+      title: i18n("UserProjectsPage.table.roleName"),
+      dataIndex: "roleName",
+      key: "roleName",
+      render: (text) => {
+        if (text === "PROJECT_USER") return i18n("projectRole.PROJECT_USER");
+        else if (text === "PROJECT_OWNER")
+          return i18n("projectRole.PROJECT_OWNER");
+      },
+    },
+    {
+      title: i18n("UserProjectsPage.table.createdDate"),
+      dataIndex: "createdDate",
+      key: "createdDate",
+      render: (text) => formatDate({ date: text }),
+    },
+    {
+      title: i18n("UserProjectsPage.table.emailSubscribed"),
+      dataIndex: "emailSubscribed",
+      key: "emailSubscribed",
+      render: (text, record) => <Switch defaultChecked={text} onChange={(checked) => updateSubscription(checked, record)} />,
+    },
+  ];
+
+  function updateSubscription(checked, record) {
+    updateProjectSubscription({ id: record.id, subscribe: checked })
+      .then((payload) => {
```

Comment:
Remove the `payload` and just replaced with `()`

Reply:
Please see 41bc802d312d8a4ecdf45ce51a28db8d2b73bfbf.